docs/_templates/template_ft_in_action.md

```diff
@@ -0,0 +1,86 @@
+# Finetuner in Action Template
+This is a template for the documentation guides of Finetuner in action, with the general structure and layout to be used for demonstrating how Finetuner can be applied to solve different tasks.
+
+```{admonition} See Also: Jina Contribution Guidelines
+:class: seealso
+For more info on best practices for documentation, see Jina's [contribution guidelines](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation)
+```
+
+## Task overview
+Describe the task which this guide accomplishes, including which model will be fine-tuned and which dataset you will use for.
+
+Also provide a brief description of what the task entails, what the dataset looks like and a high-level description of how the dataset is processed.
+
+
+## Preparing data
+Outline where the data can be found, artifact names in hubble or if relevant, how a user might load their own custom data. 
+Add a link to supplementary dataset info, for example as a `See Also` {admonition}.
+If you are outlining how to preprocess a dataset from scratch, use {dropdown} to hide long code snippets.
+
+
+## Login to Finetuner
+Explain how the user can login to Finetuner and why this is necessary. Feel free to also link to other documentation as well.
+
+Example:
+
+"As explained in the [Login to Jina ecosystem](../2_step_by_step/2_3_login_to_jina_ecosystem.md) section, first we need to login to Finetuner:"
```

Comment:
login to Jina Cloud is more tecnhically correct

Reply:
Hm ok maybe we need to change this wording in the step-by-step documentation